### PR TITLE
Extract evm version of HederaChainIdOperation

### PR DESCRIPTION
**Pull request by stoyan-lime**
**Description**:
This PR extracts a ChainId operation implementation in the evm module

**Related issue(s)**:

Fixes part of https://github.com/hashgraph/hedera-services/issues/3565

### CODE REVIEW

**Review by IvanKavaldzhiev — APPROVED** [review]
LGTM

File: hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/execution/EvmProperties.java [hunk 1]
```diff
@@ -16,10 +16,13 @@
 
 package com.hedera.node.app.service.evm.contracts.execution;
 
+import org.apache.tuweni.bytes.Bytes32;
 import org.hyperledger.besu.datatypes.Address;
 
 public interface EvmProperties {
 
+    Bytes32 chainIdBytes32();
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/execution/EvmProperties.java:24]: Need javadoc

File: hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmChainIdOperation.java [hunk 2]
```diff
@@ -14,30 +14,29 @@
...
 import static org.hyperledger.besu.evm.frame.ExceptionalHaltReason.INSUFFICIENT_GAS;
 
-import com.hedera.node.app.service.mono.context.properties.GlobalDynamicProperties;
+import com.hedera.node.app.service.evm.contracts.execution.EvmProperties;
 import javax.inject.Inject;
 import org.hyperledger.besu.evm.EVM;
 import org.hyperledger.besu.evm.frame.MessageFrame;
 import org.hyperledger.besu.evm.gascalculator.GasCalculator;
 import org.hyperledger.besu.evm.operation.AbstractOperation;
 import org.hyperledger.besu.evm.operation.Operation;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmChainIdOperation.java:28]: Need javadoc on the class

**Review by Nana-EC — APPROVED** [review]
LG but add javadocs as requested

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**